Lib/ast.py

```diff
@@ -562,15 +562,19 @@ def main():
     parser.add_argument('-m', '--mode', default='exec',
                         choices=('exec', 'single', 'eval', 'func_type'),
                         help='specify what kind of code must be parsed')
+    parser.add_argument('--no-type-comments', default=True, action='store_false',
+                        help='dont add information about type comments')
     parser.add_argument('-a', '--include-attributes', action='store_true',
                         help='include attributes such as line numbers and '
                              'column offsets')
+    parser.add_argument('-i', '--indent', type=int, default=3,
+                        help='indent level')
```

Comment:
I suggest to reuse the documentation: "indentation of nodes in AST (number of spaces)", or shorter: "indentation of nodes (number of spaces)".